ocaml/lib/pickles_bindings.ml

```diff
@@ -77,6 +77,16 @@ type pickles_rule_js =
       Js.prop >
   Js.t
 
+let map_feature_flags_option
+    (feature_flags_ : bool option Pickles_types.Plonk_types.Features.t) =
+  Pickles_types.Plonk_types.Features.map feature_flags_ ~f:(function
+    | Some true ->
+        Pickles_types.Opt.Flag.Yes
+    | Some false ->
+        Pickles_types.Opt.Flag.No
+    | None ->
+        Pickles_types.Opt.Flag.Maybe )
```

Comment:
nicely done!
btw if you'd want to model `Pickles_types.Opt.Flag.t` in TS, as an enum without contents it maps to `0 | 1 | 2`, where the order is determined by the order that `Yes | No | Maybe` appear in the enum definition